communications/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+- Give all participants as much context as possible before the meeting.
+- _Asynchronously_ resolve as many items as possible before the meeting.
+- Determine, and possibly _reduce_, the scope, and length of the meeting.
+- Determine, and possibly _eliminate_, reasons to participate in the meeting.
+- Prepare the meeting set up to be as convenient as possible for all participants.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Preferably 1 work-day before the meeting, initiate one discussion per agenda item.
```

Comment:
Recommend having two subtitles: Setting Agenda & Context for the first 2 points to make it easier. 

Also just tested out the new tool Hugo and it's pretty cool. One idea to encompass what you've written here is:
```
* Setting Agenda: Preferably 1 work-day before the meeting, initiate one discussion for agenda (instead of per agenda item)
    * Create agenda on Hugo for meeting & send to relevant slack channel or person
    * For a net-new meeting, the organizer(s) should define the initial scope of items.
    * For a recurring meeting with past sessions, use the agenda items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
    * Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
```

By sending agenda to participants in Slack from Hugo, they can start a thread in Slack and any comments on that thread will become a comment on the agenda in Hugo